queenbee/config/repositories.py

```diff
@@ -37,9 +31,7 @@ def fetch(self, auth_header: str = '') -> 'RepositoryIndex':
         from ..repository import RepositoryIndex
 
         if self.path.startswith('file:'):
-            path = self.path.split('file:')[-1]
-            index_path = os.path.join(path, 'index.json')
-            url = f'file:{index_path}'
+            url = os.path.join(self.path, 'index.json').replace('\\', '/')
```

Comment:
If we are already sanitising the path input when validating the `RepositoryReference.path` value do we still need to use `replace('\\', '/')`?

Reply:
You are right. This one is redundant knowing the the path is already cleaned. Let me remove this one.